.github/workflows/build-pkg.yml

```diff
@@ -7,22 +7,21 @@ on:
   release:
```

Comment:
Every release gets run twice: once for the tag push and once for the release publish.

https://github.com/Homebrew/brew/actions/workflows/build-pkg.yml

Might be worth making `push` jobs not run if `github.event.push.ref` starts with `ref/tags/`.

Reply:
Yeh, good catch, thanks.

> Maybe could make `push` jobs not run if `github(.event.push).ref` starts with `ref/tags/`.

Could you figure out the relevant code here?